gtda/time_series/tests/test_embedding.py

```diff
@@ -72,7 +74,27 @@ def test_embedder_transform(parameters_type, expected):
     assert_almost_equal(embedder.fit_transform(signal), expected)
 
 
+def test_embedder_resample():
+    embedder = TakensEmbedding(time_delay=4, dimension=5, stride=3)
+    embedder = embedder.fit(signal)
+    y_resampled = embedder.resample(y)
+    assert_almost_equal(y_resampled, y[np.arange(4, 20, 3)])
+
+
 def test_window_params():
-    window = SlidingWindow(width=0)
+    embedder = SlidingWindow(width=0)
```

Comment:
I'd avoid using "embedding" here, and reserve it to TakensEmbedding objects in general

Reply:
I changed to be consistent with Takens, but if clarity is a priority, I am changing it.